py_swf/clients/workflow.py

```diff
@@ -65,3 +70,341 @@ def terminate_workflow(self, workflow_id, reason):
             workflowId=workflow_id,
             reason=reason,
         )
+
+    def count_open_workflow_by_time(self, oldest_start_date, latest_start_date=None):
+        """
+        start_date should be type of datetime
+        :param oldest_start_date: datetime
+        :param latest_start_date: datetime
+        :return:
+        """
+        return self._count_open_workflow_executions(
+            oldest_start_date=oldest_start_date,
+            latest_start_date=latest_start_date,
+        )
+
+    def count_open_workflow_by_type(self, name, oldest_start_date=None, latest_start_date=None, version=None):
```

Comment:
Ah you mean like count_open_workflow_by_type_and_time() ? I think it is fine. Well I think the function name reflects the purpose or args we really need. Since time range is an optional argument, do you think it is OK to keep it and thus have a shorter name? I think the doc string may help?